### PR TITLE
Update docs-component

### PR DESCRIPTION
**Pull request by g-luo**
Update styling for documents list and display user information (nickname and email). Each item in documents list now opens that document when clicked.

### CODE REVIEW

File: project/src/main/webapp/components/user-home/docs-component.js [hunk 1]
```diff
@@ -18,26 +22,47 @@ export class DocsComponent extends LitElement {
...
-            ${
-        this.documents.map(
-            (doc) => html`
-                <li class="docs-list-element">
-                  ${doc.name} <div class="revision-text">Latest Revision: ${
-                doc.language}</div>
+            ${this.documents.map((doc) => html`
+                <li>
+                  <div>
+                    <a @click=${() => this.loadDocument(doc.hash)}>
```
Comment (max-friedman) [inline project/src/main/webapp/components/user-home/docs-component.js:50]: Nice job linking the document so it can be opened!

**Review by max-friedman — APPROVED** [review]
LGTM!

**Comment by g-luo** [comment]
image

**Review by HidyHan — APPROVED** [review]

**Review by frankpzn — APPROVED** [review]

**Review by tin0d0 — APPROVED** [review]
this is awesome! lgtm

**Pull request merged**